File: bot-api/java/src/main/java/dev/robocode/tankroyale/botapi/events/RoundStartedEvent.java
package dev.robocode.tankroyale.botapi.events;

/**
 * Event occurring when a new round has just started.
 */
@SuppressWarnings("unused")
public final class RoundStartedEvent implements IEvent {

    // The round number.
    private final int roundNumber;

    /**
     * Initializes a new instance of the RoundStartedEvent class.
     *
     * @param roundNumber is the round number.
     */
    public RoundStartedEvent(int roundNumber) {
        this.roundNumber = roundNumber;
    }

    /**
     * Returns the round number.
     *
     * @return The round number.
     */
    public int getRoundNumber() {
        return roundNumber;
    }
}

File: bot-api/python/src/tank_royale/bot_api/events/round_started_event.py
from tank_royale.bot_api.events.abstract_event import IEvent


class RoundStartedEvent(IEvent):
    """Event occurring when a new round has just started."""

    def __init__(self, round_number: int):
        """Initializes a new instance of the RoundStartedEvent class.

        Args:
            round_number: The round number.
        """
        self.round_number = round_number

    def get_round_number(self) -> int:
        """Returns the round number.

        Returns:
            The round number.
        """
        return self.round_number